root/io/io/TFileBenchmarks.cxx

```diff
@@ -0,0 +1,72 @@
+#include "TROOT.h"   // ROOT::EnableImplicitMT
+
+#include "TFile.h"	
+#include "TFrame.h"
+#include "TRandom3.h"
+#include "TBenchmark.h"
+
+#include "ROOT/RDataFrame.hxx"
+#include "ROOT/RSnapshotOptions.hxx"
+
+#include "benchmark/benchmark.h"
+#include "rootbench/RBConfig.h"
+
+
+static void BM_TFile_Write_ZLIB(benchmark::State &state) {
```

Comment:
Can you rename please "BM_TFile_Write_ZLIB" to something more explicitly saying that here we are testing Snapshot action? Maybe "BM_TFile_RDFSnapshot_ZLIB" Thanks!